### ISSUE TITLE
[DK][Rogue] Pestilence in relation with Cloak of shadows

### ISSUE THREAD

**Issue by shaxik1** [dalaranwow/dalaran-wow#4201]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
DK's Pestilence (shadow spell) http://wotlk.openwow.com/spell=51429goes through the Rogue's cloak of shadows http://wotlk.openwow.com/spell=65961 with 100% chance instead of 10%

**Expected behaviour**: Tell us what should happen instead

Pestilence should miss with 90% cahnce because it is spell and cloak of shadows decreasing the chance of hitting by 90%.
**Steps to reproduce the problem**:

1. Pestilence is not affected by Cloak of shadows
2. there is 100% to hit pestilence through the Cloak
3. It should miss with 90% cahnce

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like http://wotlk.openwow.com or http://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by Rushor** [comment]
Game/Spells: Pestilence in relation with Cloak of shadows

* closes: https://github.com/dalaranwow/dalaran-wow/issues/4201
* commit: bdb29c0eaa07d0d94442c1e4edfb8d5053b65558

The issue was fixed by a developer and will now be tested on the Public Test Realm (PTR). The changes/fixes will be applied with the NEXT changelog on the liverealm.
This means that the issue exists in game at the moment i wrote this message.
Only once the issue was marked with "Fixed - On Live Server", you can find the fix on you server.
Please watch http://www.dalaran-wow.com/news/ to see when the changelog and the fix was applied.

**Comment by robinsch** [comment]
@Rushor I ported this to PTR.

**Comment by TeridanDW** [comment]
Works as intended on PTR.

**Comment by Edzero** [comment]
This is still not working on live. Could a tester investigate it please?

**Comment by TeridanDW** [comment]
Maybe it shares the same issue with #5062 ?

> I have noticed one thing, if you press cloak while pet is casting seduction you will get charmed, if you press cloak before it casts you won't get charmed since pet wont able to cast seduction.
